v2g-liberty/rootfs/root/appdaemon/apps/v2g_liberty/v2g_app.py

```diff
@@ -60,4 +60,4 @@ async def initialize(self):
         await octopus_price_data_manager.initialize()
 
         await v2g_globals.kick_off_settings()
-        await v2g_liberty.initialise_v2g_liberty(v2g_args="initialise")
+        await v2g_liberty.kickoff_v2g_liberty(v2g_args="initialise")
```

Comment:
Do we want to stay consistent and use `kick_off_...` like on the previous line?

Reply:
true! Will do.